### PR TITLE
fix `.getAll` / utils.requestAll JSON error

### PR DESCRIPTION
**Pull request by tunnckoCore**
fix for #9

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -80,12 +80,12 @@ describe('Github API', function () {
...
     it('should concatenate all additional content with `.getAll`', function (done) {
-      this.timeout(15000);
+      this.timeout(30000);
 
       var github = new Github(creds);
-      github.get('/users/:username/repos', function (err, data) {
+      github.getAll('/users/:jon/repos', {jon: 'jonschlinkert'}, function (err, data) {
         assert.ifError(err);
-        assert.ok(data.length >= 3, 'expect `fake-user123` to have 3 or more repos');
+        assert.ok(data.length >= 500, 'expect `jonschlinkert` to have more than 500 repos');
```
Comment (jonschlinkert) [inline test.js:88]: haha nice

**Comment by doowb** [comment]
:+1: thanks @tunnckoCore 
I like the test too :grinning:

File: test.js (hunk 1 above)
Comment (tunnckoCore) [inline test.js]: em.. yea. :D Maybe it should be me (Im with 200+ only), cuz delay is too long, lol.

**Comment by tunnckoCore** [comment]
bump :100: 

@jonschlinkert @doowb

**Comment by doowb** [comment]
Thanks for the reminder.

**Pull request merged**